config.json

```diff
@@ -0,0 +1 @@
+{"applicationId":97,"deviceType":"raspberrypicm4-ioboard","userId":5,"appUpdatePollInterval":600000,"listenPort":48484,"vpnPort":443,"apiEndpoint":"https://api.cloud.nebra.com","vpnEndpoint":"vpn.cloud.nebra.com","registryEndpoint":"registry.cloud.nebra.com","deltaEndpoint":"https://delta.cloud.nebra.com","mixpanelToken":"__unused__","apiKey":"6MkBhcIUaxOvpkyEZ4KNGrj2OYLAlvxS"}
```

Comment:
cert seems to be missing.

Reply:
I've got this `config.json` from @KevinWassermann94 over Slack and it seems to have worked (at least as far as I could tell from my tests). Could you please share a file version containing everything you think is needed?

Reply:
this should be the right one. The rootca will ensure vpn authenticates. And ssh keys will ensure we can login. 

`{
    "applicationId": 130,
    "deviceType": "raspberrypicm4-ioboard",
    "userId": 2,
    "appUpdatePollInterval": 60000,
    "listenPort": 48484,
    "vpnPort": 443,
    "apiEndpoint": "https://api.cloud.nebra.com",
    "vpnEndpoint": "vpn.cloud.nebra.com",
    "registryEndpoint": "registry.cloud.nebra.com",
    "deltaEndpoint": "https://delta.cloud.nebra.com",
    "mixpanelToken": "__unused__",
    "apiKey": "H0TJRKaWVefQe0ActE2JBbRJ0yIx99k1",
    "balenaRootCA": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUZZRENDQTBpZ0F3SUJBZ0lVWmR4SzlTL1RRQ3ZkN2tua2FpSFkwS1FmYXI4d0RRWUpLb1pJaHZjTkFRRUwKQlFBd0hURWJNQmtHQTFVRUF3d1NZMkV1WTJ4dmRXUXVibVZpY21FdVkyOXRNQjRYRFRJek1ESXdNVEUwTURZegpORm9YRFRNek1ERXlPVEUwTURZek5Gb3dIVEViTUJrR0ExVUVBd3dTWTJFdVkyeHZkV1F1Ym1WaWNtRXVZMjl0Ck1JSUNJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBZzhBTUlJQ0NnS0NBZ0VBdzJ3Uyt2OTBNTkhoMHNHTUxHdC8KenhMNnJMYzlqL3IwTDMxTDJUVDlzMEJhemZoQWc5VEpydWVRZVZNOU9Fdk5Ha1RhVWwrRkJZZHlkYXZYOE1lZgpqWS9LQzVoOVV3Tm54U25pYWRqSithYTVYeHFYSTdCRGlFa25scVhiRjdXU2ZzMm5kYjM2ZTlKUjRCckg1M1V5ClhkdXl4M1FDcksvZjdHUThIQ3JLT2NtVmhJaWEwdzhMdkpaTTRQb1NPRFg1MW5ISUJ1bk5LMy9wYkJBWVhXYkIKc0pSWlRBaVhkelZrTitYT3ZUVS9BSVd5bmtDOGpmUFVwc3NxS0hlbGw3ZmpUcUJaYko5eExEU2hmY1RNTVRZUQpmR1dkT0ZLVk5LT2ExUkZwbWlDeWtNblg4T25qQTJla1BwamF0c1pkTWY1OTV4RnFGLzZ6NXdtUVY4QTE3UHFWCjdWdFZjNmJ3RTFrbkErWitXdm95SDlWU1JYUW9hbkNSL1RrQk55ZEVoaVg2UXRpNlhPLzBZS2svL2lpQkF6dXAKZk4xMDNlVjNFMmdtc1YwUVI1cWRXZXRkZ2pWaVVITWxPK1RBRXJNc2trc1k5KzJBeks1VW9Ubkpmb1UxK05SMApJS1c4OXF5MVJlbUE0YU0vWkZJTWNSK3ltdmlPakZtVStoS0l6dFBGN09kSmRKU3RGZk1ud3hYNThHMkZPV0MwCithaWRpSkZpT2I3YzhwckVLT3pHVG4yRUhyVHdYTld6enVEb1lGMUNLR2hvZEJ6amRyOU5GS2JsZDBHS3dLU04KUkJRdzUva3ZqZ2JXVEUzZTJHa1lWajBINk05eUpMdmdoanQ5SlZsSDZUdjRSZmI3ellwclREd1l0WnNBK1c1VgpvY3VsVHFkbERkWGNDbTRldDBESHRCa0NBd0VBQWFPQmx6Q0JsREFkQmdOVkhRNEVGZ1FVcmFkSmZBTkRKQUxICkEwdkpaRkkza0x2T2dXTXdXQVlEVlIwakJGRXdUNEFVcmFkSmZBTkRKQUxIQTB2SlpGSTNrTHZPZ1dPaElhUWYKTUIweEd6QVpCZ05WQkFNTUVtTmhMbU5zYjNWa0xtNWxZbkpoTG1OdmJZSVVaZHhLOVMvVFFDdmQ3a25rYWlIWQowS1FmYXI4d0RBWURWUjBUQkFVd0F3RUIvekFMQmdOVkhROEVCQU1DQVFZd0RRWUpLb1pJaHZjTkFRRUxCUUFECmdnSUJBSVV5VG9sSUhDdjQra1N2RitiajgxRFoxRmNUSWVWUjFiUWJRbW90cElRZDNBbTgvNXJFNWsyVTZQMFQKNTNOb1hLb1Q4NkRGcGpFVXZzSEF5Q2RQR3MxTXcvR2h3dVNFdFBLOFVnUHcxV05RUUwzTlQwVUE2c0x3N1RWagpvdkpnWFBnK0pRakVIN3hiMlRianpVNTdscGcrRU1mZDltdnV4eHNkNS94QTNQcGpRK1UxUy8wdmJiMlIwL08vCkIyNDNaVWJOQjJpY2dQNWdNeFVsTXdxTXIrK1R4QnhMR1BTQVB4THFIdUQwYW1uSFBCdUliampTZGdsYVhqekgKa09nUXFsdkFHMkNqTm8yb3JtN09vYkw0MjRBNzYwN1FBbW1xbFU1dkI1TkRaeUFIQzdwSG5WcHNoSGlWWHVVcwpZazB3dkpYNWx5WFRqWGZ1ZWJRRXp4ZWZjRFUxZEJ2T0pFNlBZcHlQeG1XOW1ZVTZKRitVb0lUOHUvaXU5bmtwCklBU3pIbjQyRUxCOGNiejAxUWRPRjRBZjNxTHF1ZmliMmZJUXErNmVxSFliTlEza0FmQzVUUW9oQXRObFB6VjYKZHljdzR2akkxQWpyaG14dkdrZHlNd0pnZklFdm5xSVlTYlVDbHU3Kzg2K0NXdE5CRU5rNnVHWTRmcTJrdFVpUAp4SHNaZ2xrQ2RiTkFFWmJyNnBJMkNlR2doV0djTWRiSVdBZWdDcDVDbWpGMGhNQ1lqYVdpTEtuUEwyTEI1NFdaCkU3RlFYZEpFcjk4UGFxSHRCMmdFM0R2YVplOUljNlhQNGw4T2RvMlN2cktQdFJldGtvRkUrSVllV2pxQW9uK04KMmJ4RFVCZ0xrajFOa0RPMzFoOUx1N0ZjWXRGa05oM25HNmo1UDlIMXpLRHJnVGdZCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K",
    "os": {
        "sshKeys": [
            "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDKeee0Y4o4QhVudeJ4I8Ol4idvYXlEkzxNrU1DWDkeZ"
        ]
    }
}`

Reply:
@pritamghanghas this is unrelated to this issue but do we need the root ca on an devices? As I don't think we are currently injecting that on the image generator or third party images